### DISCUSSION TITLE
Add option to overwrite existing playlist

### DISCUSSION THREAD

**Discussion by tsunulukai** [jcorporation/myMPD#1058] [Ideas]
When trying to write an updated version of a playlist, you always need to first delete the playlist, then save it again under the same name.
Trying to save it without first deleting it returns an error message:
![image](https://github.com/jcorporation/myMPD/assets/4215150/743f8ee4-da29-41ff-949a-6c2ea0ab7edd)

It would be great if myMPD could provide the option to overwrite the existing playlist and perform the delete/save operations by itself.

**Comment by jcorporation** [comment]
This is already implemented, but works only with MPD 0.24.

![image](https://github.com/jcorporation/myMPD/assets/765844/55a29867-fbd5-4044-9ba5-d08c13488969)

**Comment by tsunulukai** [comment]
My bad, works fine indeed with MPD 0.24 (beta)
Thanks !
